sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -346,21 +346,28 @@ export class KeyClient {
 
       const span = this.createSpan("createKey", unflattenedOptions);
 
-      let response: CreateKeyResponse;
+      let response: KeyVaultClientCreateKeyResponse;
 
       try {
         response = await this.client.createKey(
           this.vaultUrl,
           name,
-          keyType,
+          {
+            kty: keyType,
+            keySize: options.keySize,
+            keyOps: options.keyOps,
+            tags: options.tags
```

Comment:
The generated create key accepts curve and key type, however our general options bag in this method doesn't accept "curve". Users can still pass curve to createEcKey. I wonder, should we change the type and accept curve for the main createKey function?

cc: @heaths

Reply:
Got an answer from Heath: createKey should accept `curve`.

Reply:
Made a new issue: https://github.com/Azure/azure-sdk-for-js/issues/11263